src/core/Object3D.js

```diff
@@ -752,7 +752,7 @@ class Object3D extends EventDispatcher {
 				sphereCenter: bound.sphere.center.toArray()
 			} ) );
 
-			object.maxGeometryCount = this._maxGeometryCount;
+			object.maxInstanceCount = this._maxInstanceCount;
```

Comment:
I'm not sure how versioning works for `toJSON()`, does this require a major version bump of `metadata.version` on the outputted json?

Reply:
I would ignore this aspect for now. If someone complains, we can consider to apply a fallback on `maxGeometryCount`.